test/packages/good/kibana/dashboard/good-dashboard-abc-1.json

```diff
@@ -1,3 +1,4 @@
 {
-  "id": "good-dashboard-abc-1"
+  "id": "good-dashboard-abc-1",
+  "type": "dashboard"
```

Comment:
Why these files didn't have `type`s? Were they crafted files for tests?

Reply:
Yes I think those assets were crafted with the minimum values for the tests. At that moment, it was just the `id` field.